cli/static/generate.go

```diff
@@ -0,0 +1,22 @@
+// +build ignore
+
+package main
+
+import (
+	"log"
+
+	"github.com/linkerd/linkerd2/cli/static"
+	"github.com/shurcooL/vfsgen"
+)
+
+func main() {
+	err := vfsgen.Generate(static.Templates, vfsgen.Options{
+		Filename:     "static/generated_templates.gogen.go",
+		PackageName:  "static",
+		BuildTags:    "prod",
```

Comment:
This produces the static Go file with the `// +build prod` tag. This tag is used in the `go build -tags prod` command.